### DISCUSSION TITLE
Allow renaming "No Status" column

### DISCUSSION THREAD

**Discussion by danra** [community/community#109752] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

It would be a nice minor improvement to be able to rename "No Status" column. For example, I'd like to rename it to "Needs Triage".

Currently, I would need to either 
- add a "Needs Triage" column, requiring an extra step of moving new project issues there, and still allowing for both "No Status" or "Needs Triage" status, which isn't very clean, or
- accept the "No Status" column for the triage purpose without tweaking the name to be more descriptive.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by BlockchainSpaceDev** [comment]
Duplicate of [this one](https://github.com/orgs/community/discussions/8261)
